### ISSUE TITLE
Too many coding style violations

### ISSUE THREAD

**Issue by mudrd8mz** [ethz-let/qtype_scmc#7]
The Moodle coding style prechecker results https://moodle.org/plugins/pluginversion.php?id=19703&smurf=html report hundreds on formal coding style violations, many of them quite trivial to fix. It helps a lot with reviewing, debugging and contributing when plugins follow the common Moodle coding guidelines. Please consider fixing these.

**Comment by nexterday** [comment]
Thats true - we wanted to reserve the name in moodle plugins directory, and then cleanup (and most likely modify the plugin) to fully comply with moodle. we have many other plugins with the same case, but we have limited time for them at the moment and on the assumption that approval takes a while, we would have made the required changes by the time the approval process starts

**Comment by mudrd8mz** [comment]
Ah, right. Thanks for clarification. Shall we mark all the other plugins as needing more work too? To give your team a bit more time to deal with reported coding style issues.

**Comment by nexterday** [comment]
Yes please. we also have a major plugin coming this year (LTI qype) and this one will be submitted in full compliance (apart from external lib such as oAuth).

**Comment by mudrd8mz** [comment]
Ok then, no worries and thanks for the heads up notice. Please ping us whenever you're ready with the updated versions.

**Comment by mudrd8mz** [comment]
Just noting that a new version has been submitted for approval review in 2 March - https://tracker.moodle.org/browse/CONTRIB-7668
